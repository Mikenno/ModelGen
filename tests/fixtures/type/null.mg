
m = {"a": 1 + 2, "b": null}
pairs = [("a", 3), ("b", null), ("c", null)]

for k, v in pairs
	assert m[k] == v
	if (a = m[k]) print(k, 1, a)
	else print(k, 0)
