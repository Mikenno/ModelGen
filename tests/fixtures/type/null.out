a 1 3
b 0
c 0
